packages/lab/src/combo-box-next/useComboBox.tsx

```diff
@@ -124,6 +138,8 @@ export const useComboBox = ({
   };
 
   return {
+    value: inputValue,
```

Comment:
let's revert to `inputValue` to be aligned with Component prop, i.e. user could implement a custom combobox and see same set of names if needed.